backend/content/tests.py

```diff
@@ -1,3 +1,40 @@
 from django.test import TestCase
```

Comment:
Quick note @sulabhkatila: we also need to import `override_settings` here and in the other files :) I can get this for you post merge! 😊